lib/default.nix

```diff
@@ -94,7 +94,7 @@ in {
 
   # if it's a project package it has a src attribute set with an origSubDir attribute.
   # project packages are a subset of localPackages
-  isProjectPackage = p: p ? src && p.src ? origSubDir;
+  isProjectPackage = p: p.isProject or false;
```

Comment:
@hamishmack thinks, this might be the issue exhibited in https://hydra.iohk.io/jobset/Cardano/cardano-node-pr-2274#tabs-evaluations

Might have to revert the whole thing though.

Reply:
What, that we don't assign `isProject` in exactly the same circumstances?